dashboard/lib/services/lti.rb

```diff
@@ -81,7 +82,7 @@ def self.parse_nrps_response(nrps_response)
           sections[section_id][:members] << member
         else
           sections[section_id] = {
-            name: member_section_names[index],
+            name: "#{context_title} #{member_section_names[index]}",
```

Comment:
Thinking ahead to Scoology, it actually does this automatically, however it uses a colon, i.e. `Course 1: Section 1`. I wonder if we should match this, so we have less work for Schoology, or just deal with it at that point? 

Snippet of Schoology NRPS response below:

```
...
  "id": "https://lti-service.svc.schoology.com/lti-service/tool/7056265211/services/names-roles/v2p0/membership/7074220435",
  "context": {
    "id": "7074220435",
    "title": "Course 1: Section 2",
    "label": null
  }
...
```

Reply:
Oh good call. I was thinking we would put off adapting this for Schoology since we might refactor a lot of the sync functions, but this way we can remain consistent from the start on all the naming.